x-pack/plugin/src/test/resources/rest-api-spec/api/ml.validate.json

```diff
@@ -4,7 +4,7 @@
       "url":null,
       "description":"Validates an anomaly detection job."
     },
-    "stability":"stable",
+    "stability":"private",
```

Comment:
In light of https://github.com/elastic/elasticsearch/pull/55749#issuecomment-627289354, I think these should remain as `stable` and the url is updated to point to the ML jobs documentation: https://www.elastic.co/guide/en/machine-learning/current/ml-jobs.html